spyderlib/widgets/tests/test_array_builder.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2009- The Spyder Development Team
+# Licensed under the terms of the MIT License
+#
+
+"""
+"""
+
+# Third party imports
+from qtpy.QtCore import Qt
+from pytestqt import qtbot  # analysis:ignore
```

Comment:
Why all these `# analysis:ignore`? I don't see what doesn't follow the pep8 or other guidelines.